config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -356,3 +356,52 @@ periodics:
       options:
       - name: ndots
         value: "1"
+
+- name: aws-tests
+  interval: 6h
+  agent: kubernetes
+  decorate: true
+  # extra_refs:
+  # - org: jetstack
+  #   repo: cert-manager
+  #   base_ref: master
+  annotations:
+    testgrid-create-test-group: 'true'
+    testgrid-dashboards: jetstack-cert-manager-master
+    testgrid-alert-email: cert-manager-dev-alerts@googlegroups.com
+    description: Runs the end-to-end test suite against a EKS cluster
+  # labels:
+  #   preset-service-account: "true"
+  #   preset-dind-enabled: "true"
+  #   preset-bazel-remote-cache-enabled: "true"
+  #   preset-bazel-scratch-dir: "true"
+  #   preset-cloudflare-credentials: "true"
+  #   preset-venafi-tpp-credentials: "true"
+  #   preset-venafi-cloud-credentials: "true"
+  spec:
+    containers:
+    - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20210323-ad5071a-3.7.2
```

Comment:
How do I specify the `golang-aws` image that we built here?

Reply:
You will first need to add the infra to build that image so it gets pushed to Jetstack Build Infra images GCR and then you can get the image sha from there. So probably the commits that build the image need to go in a PR before the PR that adds this Prow job.